shared/js/background/utils.es6.js

```diff
@@ -78,12 +78,6 @@ function findParent (url) {
     }
 }
 
-function getProtocol (url) {
-    var a = document.createElement('a')
-    a.href = url
-    return a.protocol
-}
```

Comment:
Unrelated, but I decided to replace it with URL since creating DOM elements all the time is expensive